src/components/Card/Card.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { FC, Ref, useContext } from 'react';
+import { CardProps, CardSize, CardType } from './Card.types';
+import { mergeClasses } from '../../shared/utilities';
+import { ButtonShape, TwoStateButton } from '../Button';
+import { SizeContext, Size } from '../ConfigProvider';
+import { Icon, IconSize } from '../Icon';
+
+import styles from './card.module.scss';
+import { Stack } from '../Stack';
+import { Pill } from '../Pills';
+import { List } from '../List';
+
+export const Card: FC<CardProps> = React.forwardRef(
+    (
+        {
+            name,
+            icon,
+            type = CardType.list,
+            style,
+            actionButtonProps,
+            avatar,
+            size = CardSize.Medium,
+            configContextProps = {
+                noDisabledContext: false,
+                noSizeContext: false,
+            },
+            header,
+            headerButtonProps,
+            headerClassNames,
+            headerIcon,
+            body,
+            bodyClassNames,
+            bodyListOneProps,
+            bodyListTwoProps,
+            footerClassNames,
+            footerProps,
+            subHeaderProps,
+            height,
+            width,
+            ...rest
+        },
+        ref: Ref<HTMLDivElement>
+    ) => {
+        const contextuallySized: Size = useContext(SizeContext);
+        const mergedSize = configContextProps.noSizeContext
+            ? size
+            : contextuallySized || size;
+        const cardClasses: string = mergeClasses([
+            styles.card,
+            { [styles.list]: type === CardType.list },
+            { [styles.carousel]: type === CardType.carousel },
+            { [styles.cardLarge]: mergedSize === CardSize.Large },
+            { [styles.cardMedium]: mergedSize === CardSize.Medium },
+            { [styles.cardSmall]: mergedSize === CardSize.Small },
+        ]);
+
+        const headerClasses: string = mergeClasses([
+            styles.header,
+            headerClassNames,
+        ]);
+
+        const bodyClasses: string = mergeClasses([styles.body, bodyClassNames]);
+
+        const footerClasses: string = mergeClasses([
+            styles.footer,
+            footerClassNames,
+        ]);
+
+        return (
+            <div {...rest} className={cardClasses} ref={ref} style={style}>
+                <div className={headerClasses}>
```

Comment:
Should make any default content optional, this will also need to support other configurations.